vite.config.js

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import tsconfigPaths from "vite-tsconfig-paths";
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+export default defineConfig({
+  base: "",
+  root: "",
+  envPrefix: "RECORD_MANAGER_",
+  plugins: [react(), tsconfigPaths()],
+  server: {
+    port: 3000,
+  },
+  preview: {
+    port: 3000,
+  },
+  build: {
+    sourcemap: true,
+    emptyOutDir: true,
+  },
+  define: {
+    'process.env': process.env
```

Comment:
Not sure, but i believe this is workaround because of the library , right ?. 

If so, I suggest to include here a comment to explain it, referencing the library "??linked-header-??" and that it is expected to be removed if library is fixed.